projects/packages/account-protection/src/class-email-service.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Class used to define Email Service.
+ *
+ * @package automattic/jetpack-account-protection
+ */
+
+namespace Automattic\Jetpack\Account_Protection;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_Options;
+
+/**
+ * Class Email_Service
+ */
+class Email_Service {
+	/**
+	 * Send auth email.
+	 *
+	 * @param WP_User $user The user.
+	 * @param string  $auth_code The authentication code.
+	 * @return bool True if the email was sent successfully, false otherwise.
+	 */
+	public function send_auth_email( $user, $auth_code ): bool {
+		$wp_send = $this->wp_send_auth_email( $user, $auth_code );
+
+		if ( ! $wp_send ) {
+			$api_send = $this->api_send_auth_email( $user, $auth_code );
+
+			return $api_send;
```

Comment:
nitpick: a bit redundant
```suggestion
			return $this->api_send_auth_email( $user, $auth_code );
```